feature_engine/creation/mathematical_combination.py

```diff
@@ -61,6 +61,9 @@ class MathematicalCombination(BaseEstimator, TransformerMixin):
         contain missing values. If 'ignore', missing data will be ignored when
         performing the calculations.
 
+    drop_original: bool, default=False
+        If True, the original variables to transform will be dropped from the dataframe.
```

Comment:
I would change the wording to:

If True, the original variables will be dropped from the dataframe after their combination.